Gems/Atom/Feature/Common/Assets/Materials/Types/StandardPBR_LowEndForward.shader

```diff
@@ -5,7 +5,9 @@
     // DrawListTag. If your pipeline doesn't have a "lowEndForward" DrawListTag, no draw items
     // for this shader will be created.
 
-    "Source" : "./StandardPBR_LowEndForward.azsl",
+    "Source" : "./StandardPBR_ForwardPass.azsl",
+
+    "Definitions": [ "QUALITY_LOW_END=1" ],
```

Comment:
Awesome, didn't know this functionality was added already! Very excited to start using it

Reply:
Yea I added it for this change lol